### PR TITLE
Add getEmailsByToAddress Function

### PR DESCRIPTION
**Pull request by Regan831**

### CODE REVIEW

File: lib/mailserver.js [hunk 1]
```diff
@@ -468,4 +468,48 @@ mailServer.relayMail = function(idOrMailObject, specifiedEmailRelays, done) {
...
+                }
+            },
+            "sort": [
+                {"time": {"order": "desc"}}
+            ]
+        }
+    }).then(
+        function (response) {
+            emails.totalPossible = response.hits.total;
+            response.hits.hits.forEach(function(hit) {
```
Comment (sarahleeashraf) [inline lib/mailserver.js]: `hits.hits`?

Comment (mgarlanger) [inline lib/mailserver.js]: yes, that is what comes back from ES for this type of query `hits.hits`

Comment (Regan831) [inline lib/mailserver.js]: Yeah, that is the format of the ES response.
https://github.com/signpost/MailDev/blob/integrate_with_es/lib/mailserver.js#L298

Comment (Regan831) [inline lib/mailserver.js]: https://search-fusion-maildev-icj2q2zqnrcjb67qwjhqiyk6sq.us-east-1.es.amazonaws.com/emails/email/_search

File: lib/mailserver.js [hunk 2]
```diff
@@ -468,4 +468,48 @@ mailServer.relayMail = function(idOrMailObject, specifiedEmailRelays, done) {
...
+                var data = hit._source;
```
Comment (mgarlanger) [inline lib/mailserver.js]: unneeded variable.

File: lib/web.js [hunk 3]
```diff
@@ -55,13 +43,11 @@ app.get('/emails/email/search', function(req, res){
 
 });
 
-// Delete all emails
-app.delete('/email/all', function(req, res){
-
-  mailserver.deleteAllEmail(function(err){
-    if (err) return res.status(500).json({ error: err.message });
+app.get('/emails/email/searchBy', function(req, res){
```
Comment (sarahleeashraf) [inline lib/web.js]: looks like we're missing a test for this?

Comment (Regan831) [inline lib/web.js]: oh yeah, there currently isn't a test for web.js. I can start one

**Pull request merged**